.circleci/config.yml

```diff
@@ -0,0 +1,113 @@
+version: 2.1
+
+executors:
+  my-executor:
+    docker:
+      - image: continuumio/miniconda3
+      # astroconda/python:3.6.1
+    working_directory: ~/repo
+
+jobs:
+  build:
+    executor: my-executor
+    environment:
+      - SOURCE_BRANCH: master
+      - TARGET_BRANCH: gh-pages
+    steps:
+      - add_ssh_keys
+
+      - checkout
+
+      - restore_cache:
+          key: v1-miniconda-{{ .Branch }}
+
+      - run:
+          name: Install miniconda
+          command: |
+            ./.circleci/install_conda.sh
+
+      - save_cache:
+          key: v1-miniconda-{{ .Branch }}
+          paths:
+            - /home/circleci/miniconda
+
+      - restore_cache:
+          key: v1-miniconda-depends-{{ .Branch }}
+
+      - run:
+          name: Setup Environment
+          command: |
+            ./.circleci/setup_env.sh
+
+      - save_cache:
+          key: v1-miniconda-depends-{{ .Branch }}
+          paths:
+            - /home/circleci/miniconda
+
+      - run:
+          name: Run/convert notebooks
+          command: |
+            source activate /home/circleci/miniconda/envs/notebooks_env
+            conda info --envs
+            python ./convert.py
+
+      - run:
+          name: Check notebooks
+          command: |
+            COMMIT_RANGE=$(echo "${CIRCLE_COMPARE_URL}" | cut -d/ -f7)
```

Comment:
Is there somewhere that says clearly just what `CIRCLE_COMPARE_URL` actually is?  I know the travis version of this is a bit questionable in that it can be buggy what it decides is the "base" SHA for any given build...

Reply:
[Its stated here as the URL for the repo to compare commits of the build](https://circleci.com/docs/2.0/env-vars/) 
I also found this as an alternative to Travis CI commit range, its a feature request but this is a [solution](https://discuss.circleci.com/t/get-list-of-commits-in-build/15725) for now in CircleCI